x-pack/plugins/uptime/server/lib/requests/get_pings.ts

```diff
@@ -84,6 +85,23 @@ export const getPings: UMElasticsearchQueryFn<GetPingsParams, PingsResponse> = a
       : {}),
   };
 
+  // if there are excluded locations, add a clause to the query's filter
+  const excludedLocationsArray =
+    typeof excludedLocations !== 'undefined' ? JSON.parse(excludedLocations) : undefined;
```

Comment:
Would it be possible to do 
```
const excludedLocationsArray = excludedLocations && JSON.parse(excludedLocations);
```

Reply:
Very possible. bded0bb577964c1feb4607d09515208c313f11e0

I also added an explicit string check; LMK if you think that is excessive.